src/Illuminate/Database/Eloquent/Concerns/GuardsAttributes.php

```diff
@@ -151,8 +149,7 @@ public function isFillable($key)
             return false;
         }
 
-        return empty($this->getFillable()) &&
-            ! Str::startsWith($key, '_');
+        return empty($this->getFillable());
```

Comment:
Why make it breaking though?

    return empty($this->getFillable()) && ! $this->shouldBeGuarded($key);

    protected function shouldBeGuarded($key)
    {
        return Str::startsWith($key, '_');
    }

You can then override `shouldBeGuarded` for your models if you don't want this specific guarding logic.

Reply:
Yes, and if this is a good change I like to propose it for everyone so others may benefit, including me in my other Laravel projects. If not a good change, I am sure Taylor will reject this :)